tools/c7n_azure/c7n_azure/session.py

```diff
@@ -62,21 +62,21 @@ def __init__(self, subscription_id=None):
             )
             self.subscription_id = os.environ['AZURE_SUBSCRIPTION_ID']
             self.tenant_id = os.environ['AZURE_TENANT_ID']
-            self.log.info("Creating session with Service Principal Authentication")
+            self.log.debug("Creating session with Service Principal Authentication")
 
         else:
             # Azure CLI authentication
             (self.credentials,
              self.subscription_id,
              self.tenant_id) = Profile().get_login_credentials(
                 resource=AZURE_PUBLIC_CLOUD.endpoints.active_directory_resource_id)
-            self.log.info("Creating session with Azure CLI Authentication")
+            self.log.debug("Creating session with Azure CLI Authentication")
 
         # Let provided id parameter override everything else
         if subscription_id is not None:
             self.subscription_id = subscription_id
 
-        self.log.info("Session using Subscription ID: %s" % self.subscription_id)
+        self.log.debug("Session using Subscription ID: %s" % self.subscription_id)
```

Comment:
This one should probably stay at info, along with denoting Azure Session. Multi subscription management is much more common, and our mechanism of subscription acquisition is more implicit given the common culture.

Reply:
I see. I suppose the main gist of this PR was to prevent logging during commands that weren't "run", here session information is being logged during a schema command. I'll look at if there's a way to prevent excess logging during that then.